app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -281,13 +281,15 @@ export class InviteUserModal extends Component {
                 format={this.formatValueProject}
                 parse={this.parseValueProject}
               >
-                <InputTagsSearch
-                  minLength={1}
-                  async
-                  uri={this.projectSearchUrl}
-                  makeOptions={this.formatValue}
-                  filterOption={this.filterProject}
-                />
+                <FieldErrorHint>
```

Comment:
Add `hintType="top"` too for avoiding overlapping search results.

Reply:
Fixed